src/resmom_win/start_exec.c

```diff
@@ -2021,6 +2021,13 @@ finish_exec(job *pjob)
 	mom_hook_input_init(&hook_input);
 	hook_input.pjob = pjob;
 	if (do_tolerate_node_failures(pjob)) {
+		char	err_msg[LOG_BUF_SIZE];
```

Comment:
This is a case where Codacy will NOT complain if you declare err_msg at the top level of the function because you end up using it in two places. Not sure if the compiler is smart enough to recognize that or not.

Reply:
Ok, I'll declare err_msg at the top level of the function.